cva6/regress/dv-riscv-arch-test.sh

```diff
@@ -19,13 +19,13 @@ source ./cva6/regress/install-riscv-dv.sh
 source ./cva6/regress/install-riscv-arch-test.sh
 
 if ! [ -n "$DV_TARGET" ]; then
-  DV_TARGET=cv64a6_imafdc_sv39
+  DV_TARGET=cv32a60x
```

Comment:
Do not upstream these changes, these changes are only supposed to be done for local testing.